src/palace/manager/api/admin/controller/library_settings.py

```diff
@@ -183,6 +191,149 @@ def create_library(self, short_name: str) -> tuple[Library, bool]:
         )
         return library, is_new
 
+    def import_libraries(self) -> Response | ProblemDetail:
```

Comment:
Since we removed `self.require_system_admin` above, and I don't see anywhere we are validating the admins permissions on the endpoint, wouldn't this endpoint allow any logged in user to create libraries? I don't think this is what we want. It seems like this should be an admin only level function.

I think this needs to get fixed before we can merge this.